include/zephyr/pm/policy.h

```diff
@@ -196,10 +202,12 @@ void pm_policy_latency_changed_unsubscribe(struct pm_policy_latency_subscription
  *
  * @param evt Event.
  * @param time_us When the event will occur, in microseconds from now.
```

Comment:
Now I think this register's precision can be insufficient for low-latency events. It defines relative time as "from now", but "now" is an imprecise term. For low-latency events with strict timings, it would be better to use a precise absolute time unit.

Reply:
Would it be possible to align this parameter with `cycle` from the low-latency variant? Having two almost identical functions taking different units to express time is hard to use and error-prone.

Reply:
Do you mean modifying this function to use as well absolute time ?

Reply:
Yes, I think the API would be best if both functions used the same units. We need absolute for the precision of low-latency API, so both should use absolute.